app/src/main/java/com/codesoom/assignment/Status.java

```diff
@@ -0,0 +1,18 @@
+package com.codesoom.assignment;
+
+public enum Status {
```

Comment:
`enum`으로 분리하니 보기 좋습니다. 그런데 이름은 아쉽습니다. 의미하는 바가 평범한 `상태`가 아니라 `HttpStatus`이기 때문입니다.
```suggestion
public enum HttpStatus {
```

[RFC2616]( https://tools.ietf.org/html/rfc2616#section-10 )의 Http Status 섹션도 읽어보시길 권합니다. (RFC2616은 HTTP를 처음 만든 팀 버너스 리가 직접 작성한 문서입니다.)